libs/secure-sm/src/main/java/org/elasticsearch/secure_sm/SecureSM.java

```diff
@@ -106,7 +106,9 @@ public static SecureSM createTestSecureSM() {
         // eclipse test runner
         "org\\.eclipse.jdt\\.internal\\.junit\\.runner\\..*",
         // intellij test runner
-        "com\\.intellij\\.rt\\.execution\\.junit\\..*"
+        "com\\.intellij\\.rt\\.execution\\.junit\\..*",
+        // intellij test runner (newer IDEA versions)
```

Comment:
Can you please add the version of IDEA this starts with, so we can understand when the older one can be removed (ie when older versions are no longer relevant)?